connector_magento/components/backend_adapter.py

```diff
@@ -153,9 +153,20 @@ def call(self, method, arguments, http_method=None, storeview=None):
             # record(method, arguments, result)
             return result
         except (socket.gaierror, socket.error, socket.timeout) as err:
-            raise NetworkRetryableError(
-                'A network error caused the failure of the job: '
-                '%s' % err)
+            if err.response is not None:
```

Comment:
You can use `if err.response:` here `err.resonse` is no use to us if it is an empty string or a boolean False etc.

Reply:
I have checked the type of the response and it is `<class 'requests.models.Response'>` , using only `if err.response:` didn't work when I was testing.

Reply:
Oh, in that case my comment is wrong and your code is spot on. Thanks for clarifying!